package.json

```diff
@@ -7,7 +7,8 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "ng e2e"
+    "e2e": "ng e2e",
+    "extract": "ngx-translate-extract --input ./src --output ./src/assets/i18n/en.json ./src/assets/i18n/es.json --clean --sort --format namespaced-json --marker _"
```

Comment:
not sure if we actually need this script and dependency as you already extracted all the possible strings to be translated and going forward every string should have a translation??

Reply:
This script lets you keep updating the json files when you add new labels in html files. It is a helper.
E.g. In home.component.html you add` {{ 'home.test' | translate }}`
then you run: `npm run extract` and i18n json files will be updated.

Reply:
ok sounds good. Never tried, but seems that make sense put them in spanish and then move it to translations automatically :)